hoodie-common/src/main/java/com/uber/hoodie/common/spark/HoodieSparkTaskContext.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2018 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.hoodie.common.spark;
+
+import org.apache.spark.TaskContext;
+import scala.Serializable;
+
+public class HoodieSparkTaskContext implements Serializable {
```

Comment:
don't understand the motivation for this class.. https://spark.apache.org/docs/2.2.0/api/java/org/apache/spark/TaskContext.html seems serializable and all this is doing is picking out four variables out of the TaskContext class. Can we remove this.

Reply:
Also like I mentioned this class should not reside inside hoodie-common anyways

Reply:
removed this class. setting it via "TaskContext$.MODULE$.setTaskContext(taskContext);"